stac_fastapi/types/setup.py

```diff
@@ -10,6 +10,8 @@
     "attrs",
     "pydantic[dotenv]",
     "stac_pydantic==2.0.*",
+    "pystac==1.*",
```

Comment:
it took me 2 hours to figure out that the docs broke because I'd added an import from pystac, but it's not a dependency of this package. The tests worked because everything gets installed when they run.